protos/camera/camera.proto

```diff
@@ -311,6 +320,18 @@ message VideoStreamInfo {
     Status status = 2; // Current status of video streaming
 }
 
+// Type for video stream status. Difference to VideoStreamSettings: Status gets sent whenever there is a change, e.g. fov of the camera. Settings get queried usually once during startup.
+message VideoStreamStatus{
+    bool is_running = 1; // Video stream is running / not running
+    bool is_thermal = 2; // Indicates whether the stream is from thermal imaging (true) or visible light (false)
+    float frame_rate_hz = 3; // Frames per second
+    uint32 horizontal_resolution_pix = 4; // Horizontal resolution (in pixels)
+    uint32 vertical_resolution_pix = 5; // Vertical resolution (in pixels)
+    uint32 bit_rate_b_s = 6; // Bit rate (in bits per second)
+    uint32 rotation_deg = 7; // Video image rotation (clockwise, 0-359 degrees)
+    uint32 horizontal_fov_deg = 8; // Horizontal field of view in degrees
```

Comment:
These two types we should change to `float` anyway, either here or in `VideoStreamSettings` or both.

Reply:
`horizontal_fov_deg` changed to float. keeping `rotation_deg` to avoid breaking existing code